### ISSUE TITLE
globally disable automatic keyword discovery 

### ISSUE THREAD

**Issue by DetachHead** [robotframework/robotframework#4439]
it's annoying having to put `ROBOT_AUTO_KEYWORDS = False` in every python file. it would be cool if there was an option to disable it globally.

**Comment by pekkaklarck** [comment]
Yeah, `ROBOT_AUTO_KEYWORDS = False` is a bit ugly. Would it be better if there would some way to use the `@library` decorator that works with classes also with modules? AFAIK Python doesn't have syntax for decorating modules, but we could support something like
```python
from robot.api.deco import library

ROBOT_LIBRARY = library()
```
and then on Robot side decorate the module with that `ROBOT_LIBRARY`. This approach would have a benefit that you could at the same time specify custom converters, version, and so on.

**Comment by KotlinIsland** [comment]
I have found that there are issues when one module will import a keyword from another module, this causes the same keyword to be discovered twice under different names and disabling automatic keyword discovery will have no effect on this scenario.
